src/styles/utils/variables.scss

```diff
@@ -0,0 +1,9 @@
+$card-wigth: 200px;
```

Comment:
This file better name with '_'  in beginning.

More about naming and file structure in scss in this [video, timing 1:00](https://mate.academy/learn/css-advanced/sass#/video/931)